flake.nix

```diff
@@ -18,7 +18,7 @@
     services-flake.url = "github:tgunnoe/services-flake";
 
     # Sidechains deps
-    trustless-sidechain.url = "github:input-output-hk/trustless-sidechain/v6.0.0-rc3";
+    trustless-sidechain.url = "github:input-output-hk/partner-chains-smart-contracts/692e7d2444229962b922e98a3eb16345f7d06f4c";
```

Comment:
i'd wait for a proper tag

Reply:
Yeah, I will

Reply:
I don't know if we can convince SETs to sign off smart-contracts w/o having partner-chains-cli tests. Don't shoot messenger. I know who is right.